cg/meta/workflow/fluffy.py

```diff
@@ -20,6 +28,51 @@
 LOG = logging.getLogger(__name__)
 
 
+class FluffySampleSheetHeaders(Enum):
+    flow_cell_id: str = "FCID"
+    lane: str = "Lane"
+    sample_internal_id: str = "Sample_ID"
+    sample_reference: str = "SampleRef"
+    index: str = "index"
+    index2: str = "index2"
+    sample_name: str = "SampleName"
+    control: str = "Control"
+    recipe: str = "Recipe"
+    operator: str = "Operator"
+    sample_project: str = "SampleProject"
+    exclude: str = "Exclude"
+    library_nM: str = "Library_nM"
+    sequencing_date: str = "SequencingDate"
+
+
+class FluffySampleSheetEntry(BaseModel):
```

Comment:
to make it consistent with the other sample sheets
```suggestion
class FluffySampleSheetSample(BaseModel):
```

Reply:
Renamed it to `FluffySample`